### ISSUE TITLE
2.0.13: cannot find module './api' from 'node_modules/mdn-data/index.js'

### ISSUE THREAD

**Issue by mrtnzlml** [mdn/data#442]
Hi! 👋 I am trying to upgrade 2.0.12 to 2.0.13 but I am getting the following error:

```text
Cannot find module './api' from 'node_modules/mdn-data/index.js'
```

I checked what's different between the versions and there are no `index.js` files inside `/api`, `/css` and `/I10n` (therefore the main `index.js` fails to require them). You can see the failure here: https://github.com/adeira/universe/pull/1310

Can you please have a look at what went wrong during the publishing? Or are we doing something wrong? Either way, this looks like a breaking change that should not be in the patch version. 🤔 

Thanks!

**Comment by chrisdavidmills** [comment]
Hi @mrtnzlml !

I'm sorry to hear that you're having trouble upgrading this. I've done checks on our side, and it seems fine, all the checks passing, etc.

I wonder if you tried to do this mid update? Is it still failing for you?

**Comment by mrtnzlml** [comment]
@chrisdavidmills Here are steps to reproduce:

```text
yarn add mdn-data@2.0.13
echo "require('mdn-data')" > index.js
node index.js
```

It throws the following error:

```text
Error: Cannot find module './api'
```

It works OK when you use version 2.0.12:

```
yarn add mdn-data@2.0.12
node index.js
```

Am I doing something wrong? Here is what's different:

```
yarn add mdn-data@2.0.13
ls node_modules/mdn-data/api 

inheritance.json        inheritance.schema.json
```

vs.

```
yarn add mdn-data@2.0.12
ls node_modules/mdn-data/api 

index.js                inheritance.json        inheritance.schema.json
```

Notice the missing `index.js` in version 2.0.13 🤔

**Comment by escattone** [comment]
@mrtnzlml I can confirm what you're seeing as well. We're using `mdn-data` to build our documents on MDN and version 2.0.13 is failing for us as well. I'm not familiar with `mdn-data`, but I'll look into it right now. Thanks for letting us know.

**Comment by lahmatiy** [comment]
Didn't see this issue, but I fixed it :)
https://github.com/mdn/data/pull/443

**Comment by escattone** [comment]
A new version 2.0.14 is now available that resolves this issue.

Thanks again @lahmatiy for your fix (#443) and @mrtnzlml for reporting the issue. This was due to Travis starting to use Node 15 for their `stable` release, which in turn pulled in `npm` version 7, which in turn, unlike `npm` version 6, strictly respects the `files` property of the `package.json` file.
